### PR TITLE
Create developer-certificate.md, fixed issue #560 for Hacktoberfest

### LINKED ISSUE #560: [DOC] - Developer Certificate of Origin
### What is incorrect in the documentation?

The Developer Certificate of Origin is currently not present in the contributing documentation.

### What should that text say?

A new file should be added in the `docs` directory called `developer-certificate.md`.

The text within should be the text of the [Developer Certificate of Origin](https://developercertificate.org/).

### Additional information

Must be done before https://github.com/BeccaLyria/discord-bot/issues/869

### PR DESCRIPTION
**Pull request by KanhaKorgaonkar**
# Pull Request
Documentation change that fixes #560 

## Description:

Added a Developer Certificate and created a new markdown file for the same.

## Related Issue:

Closes #560

### CODE REVIEW

File: docs/developer-certificate.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+(c) The contribution was provided directly to me by some other
+    person who certified (a), (b) or (c) and I have not modified
+    it.
+
+(d) I understand and agree that this project and the contribution
+    are public and that a record of the contribution (including all
+    personal information I submit with it, including my sign-off) is
+    maintained indefinitely and may be redistributed consistent with
+    this project or the open source license(s) involved.
```
Comment (nhcarrigan) [inline docs/developer-certificate.md:34]:
````suggestion
# Developer Certificate of Origin

All contributions to Becca Lyria must comply with this Developer Certificate of Origin:

```md
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```
````

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
Added a suggestion to tweak the formatting.

Additionally, could you add a link to this in the `_navbar.md` file? It should come directly after the `- [How to Contribute](/how-to-contribute.md)` line.

**Review by nhcarrigan — APPROVED** [review]
This looks good to me!

**Comment by nhcarrigan** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

**Comment by KanhaKorgaonkar** [comment]
Thank you, @nhcarrigan. Can you please add the Hacktoberfest-accepted tag too?

**Comment by KanhaKorgaonkar** [comment]
Thanks!

**Comment by nhcarrigan** [comment]
The `hacktoberfest-accepted` tag is not necessary because this project has the `hacktoberfest` topic. 🙂

**Comment by KanhaKorgaonkar** [comment]
> The `hacktoberfest-accepted` tag is not necessary because this project has the `hacktoberfest` topic. 🙂

Yeah I got that, thanks a lot! This is my first year contributing, so a little unfamiliar with this.

**Pull request merged**